### PR TITLE
MMT-3556: As a user, I want to see a Mange Association tab

### PR DESCRIPTION
**Pull request by dmistry1**
During SIT testing Francell found these following issues: 

1. When the user Edits a published Variable record that has a pre-existing collection association, the association between the Collection record and the Variable record no longer exits. Is this the expected behavior?

1. When the user selects the “Manage Collection Associations” button for a published Variable record with its pre-existing collection association, and then updates the collection association, Edits the Variable Record (accessed through the Breadcrumb), and then attempts to Publish the Variable draft, the progress circle spins continuously

**To reproduce**
- Search for a published Variable record
- Click on the “Manage Collection Associations” button
- Click on the “Update Collection Association” button
- Search for a Collection
- Click on the “Create Association” button for one of the collections returned
- Click on the Variable record link in the breadcrumb
- Click on the “Edit Variable Record” link
- Click on the “Publish Variable Draft” button
- The progress wheel just continues to spin

These two issues have been resolved and verified with Francell locally.

### CODE REVIEW

File: static/src/js/hooks/usePublishMutation.js [hunk 1]
```diff
@@ -1,55 +1,51 @@
...
-  const { addNotification } = useNotificationsContext()
+  const { conceptId } = useParams()
 
-  const publishMutation = (conceptType, nativeId, collectionConceptId) => {
-    publishDraftMutation({
+  const publishMutation = useCallback(async (
+    conceptType,
+    nativeId,
+    collectionConceptId
+    // ConceptId
```
Comment (mandyparson) [inline static/src/js/hooks/usePublishMutation.js:21]: remove

**Review by mandyparson — COMMENTED** [review]
Screenshot 2024-04-09 at 11 20 00 AM
The breadcrumb in SIT is plural, just a FYI. Also not 'Showing 0 Collection Association' (should be associationS or specific use case error).

**Comment by tle1989** [comment]
Singular in Manage Collection Association in SIT MMT
Screenshot 2024-04-09 at 1 27 26 PM

**Comment by tle1989** [comment]
Breadcrumb for Collection Association, and Variable Record Collection Associations are opposite to SIT, and, does not have"Each variable can only be associated with a single collection", and table header and row shaded area are also opposite to SIT
Screenshot 2024-04-09 at 1 36 37 PM

**Comment by tle1989** [comment]
Update Collection Associations with s is opposite to SIT
Screenshot 2024-04-09 at 1 44 24 PM

**Comment by tle1989** [comment]
Update Collection Association button --> singular in SIT 
Screenshot 2024-04-09 at 1 58 41 PM

**Review by tle1989 — COMMENTED** [review]
Please see the screenshots!

**Review by tle1989 — APPROVED** [review]

**Review by mandyparson — APPROVED** [review]

**Pull request merged**